src/cli/options.ts

```diff
@@ -54,3 +54,14 @@ export function withStateOption<T>(
     once: true,
   });
 }
+
+export function withGenerateFormattedCodeSnippetsOption<T>(
+  yargs: Argv<T>
+): Argv<T & { generateFormattedCodeSnippets?: string }> {
+  return option(yargs, "format", {
```

Comment:
Which format? Sphynx style rST is not the only way this could be done. That said, I like the idea of potentially accepting a generic "format" flag and having "sphynx-literalinclude" be one of possible values for it.

Reply:
That's the plan, that you'll feed in "--format=rst" in the first pass and we can always add more in the future.